packages/dx-react-grid/docs/reference/table.md

```diff
@@ -58,69 +59,69 @@ rowId? | number &#124; string | Specifies the associated row's ID.
 row? | any | Specifies the associated row.
 height? | number | Specifies the table row height.
 
-### <a name="table-column"></a>TableColumn
+### TableColumn
 
-Describes table column properties that the Table plugin takes into account.
+Describes properties of a table column the Table plugin renders.
 
 A value with the following shape:
 
 Field | Type | Description
 ------|------|------------
 key | string | A table column's unique identifier.
 type | string | Specifies the table column type. The specified value defines which cell template is used to render the column.
-column? | [Column](#column) | Specifies the associated user column.
+column? | [Column](#column-extension) | Specifies the associated user column.
```

Comment:
It looks like you removed all '-' symbols from link anchors, but not here.

Reply:
It's a way markdown generate default header ids.